data_graph/views.py

```diff
@@ -0,0 +1,34 @@
+'''Provide API views for data-graph module.'''
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+
+from data_graph.services import DataGraphService, DataGraphParamsService
+
+
+class DataGraphView(APIView):
+    '''create API views for getting graph data'''
+
+    def get(self, request):
+        '''getting data-graph data'''
+        try:
+            request_data = {
+                'y_type': int(request.GET.get('y_type')),
```

Comment:
I don't think it wise to hard code parameters here, if we try to support other graph but with different parameters settings, then we stuck at here because the parameters are determined and not flexible. Consider just extract the `graph_type` here and leave other `request.GET` parameters as a dict, the real method should be responsible for preparing the data based on extracted parameters, not the view here.